src/redux/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import bookReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  books: bookReducer,
+  categories: categoriesReducer,
+});
+
+const store = createStore(
+  rootReducer,
+  applyMiddleware(logger),
+);
+
```

Comment:
- [ ] @momotsow great work in creating the store, however as the previous reviewer pointed out its a project requirement to use the redux toolkit instead of redux to create the store 
![image](https://user-images.githubusercontent.com/61408860/207806010-a657b052-c511-4c17-9d3a-fd4ca895d98c.png)

I am also getting the error below when I run the app, it would be nice if you ensured all modules are istalled correctly
![image](https://user-images.githubusercontent.com/61408860/207806431-b0518a83-0461-4431-926b-8a18edbb0e74.png)